Islandora-CLAW/islandora
# Linked agents

**GitHub Issue**: Part of https://github.com/Islandora-CLAW/CLAW/issues/905

* Goes with https://github.com/Islandora-Devops/migrate_7x_claw/pull/10

# What does this Pull Request do?

Exports config for downloading an original file when viewing binaries.  It's part of the changes for https://github.com/Islandora-CLAW/CLAW/issues/905, where we're tagging OBJs as Original Files instead of Preservation Masters


# How should this be tested?

Check out the testing instructions on https://github.com/Islandora-Devops/migrate_7x_claw/pull/10

# Interested parties
Tag (@ mention) interested parties or, if unsure, @Islandora-CLAW/committers
